scripts/usb_reset.py

```diff
@@ -200,7 +200,23 @@ def deny_device(path, domid):
     _device_ctl(path, domid, False)
 
 
-def setup_cgroup(domid, pid):
+def setup_cgroup(domid, pid):  # type:(str, str) -> None
```

Comment:
Yep, I added `print(type(domid),type(pid))` there and it prints `<class 'int'> <class 'int'>`:
```suggestion
def setup_cgroup(domid, pid):  # type:(int, int) -> None
```
As this is only a type comment for static type checkers, the change in the type comment would have an effect if we'd make a change that would be dependent on the type. Right now, the code is fully agnostic to the type being `int` or `str`, therefore no type checker had reason to complain about the comment. While not correct, it wasn't really a big issue at all.

Reply:
> Is this comment a special syntax for mypy? Just wondering why not use the following as type hints:
> 
> ```python
> def setup_cgroup(domid:int, pid: int) -> None:
> ...
> ```
Type hints introduced in Python 3.5 as a part of the standard library via PEP 484, it can't pass the Python2 unittest in github action and `%py_byte_compile` (python2) in xapi.spec.
So we need to keep Py2 + Py3 compatible until we complete the full migration to Python3.